Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -323,3 +355,32 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 	}
 	return utilerrors.NewAggregate(allErrs)
 }
+
+// mustPrintWithKinds determines if printer output
+// should print multiple kinds of objects, even if
+// "all-kinds" flag is not set
```

Comment:
Instead, it would be nice to explain *why* this function will return true, under which circumstance it does so.

(And the answer is when there are objects of multiple kinds, right?).